src/Modal/Modal.js

```diff
@@ -41,6 +41,7 @@ function CenterOrFullscreen({
 	fullscreen,
 	spacing,
 	width,
+	disableIE11Hack
```

Comment:
I'm using trailing commas everywhere in base5-ui 😅 

```suggestion
	disableIE11Hack,
```

Reply:
one step ahead of you, the linter told me off